src/test/system_tests/ceph_s3_tests/test_ceph_nsfs_s3_config_and_run_s3_tests.sh

```diff
@@ -0,0 +1,17 @@
+# This script objective is to run Ceph s3 tests
+# We use it inside of K8S job, see noobaa-tests-s3-job
+# since we use "|| true" this job will always end with "Completed" status
+
+#!/bin/bash
+set -x
+
+CEPH_S3_DIRECTORY="src/test/system_tests/ceph_s3_tests"
+CEPH_S3_TESTS_CONFIG="${CEPH_S3_DIRECTORY}/test_ceph_nsfs_s3_config_setup.js"
+CEPH_S3_RUN_TESTS="${CEPH_S3_DIRECTORY}/test_ceph_s3.js"
+S3_CEPH_TEST_BLACKLIST="${CEPH_S3_DIRECTORY}/s3-tests-lists/nsfs_s3_tests_black_list.txt"
+S3_CEPH_TEST_PENDING_LIST="${CEPH_S3_DIRECTORY}/s3-tests-lists/nsfs_s3_tests_pending_list.txt"
+NUMBER_OF_WORKERS=1
```

Comment:
Did you try to change the number of workers?
We had problems with the old configuration using minikube on the CI and tried to run it without concurrency, maybe you can increase it and check: (1) see if it works (2) see if it decreases the duration.

Reply:
I will try that

Reply:
JOBS > 1 has concurrency issue. Not working.